lib/capistrano/slack_notify.rb

```diff
@@ -96,11 +96,15 @@ def repository
     end
 
     def rev
-      @rev ||= `git ls-remote #{repository} #{branch}`.split(" ").first
+      @rev ||= `git ls-remote #{repository} #{branch_or_not}`.split(" ").first
+    end
+
+    def branch_or_not
+      fetch(:branch, '')
     end
 
     def deploy_target
-      [slack_app_name, branch].join('/') + (rev ? " (#{rev[0..5]})" : "")
+      [slack_app_name, branch_or_not].join('/') + (rev ? " (#{rev[0..5]})" : "")
```

Comment:
This will produce `slack_app_name/@revision` when a branch name isn’t specified. Are you cool with this or would you rather see `slack_app_name@revision`?

Reply:
Oh that's a good point. No, I think seeing `slack_app_name@revision` would be better. I'll update the PR.